adapter/lldb/src/sb/sbaddress.rs

```diff
@@ -2,7 +2,7 @@ use super::*;
 
 cpp_class!(pub unsafe struct SBAddress as "SBAddress");
 
-unsafe impl Send for SBAddress {}
```

Comment:
AFAICT, none of the `SB*` structs should be `Send`. Everything continues working with these removed. Let me know what you think @vadimcn.